python/ray/_private/accelerators/amd_gpu.py

```diff
@@ -54,32 +53,35 @@ def get_current_process_visible_accelerator_ids() -> Optional[List[str]]:
 
     @staticmethod
     def get_current_node_num_accelerators() -> int:
+        import ray._private.thirdparty.pyamdsmi as pyamdsmi
+
         num_gpus = 0
 
-        if sys.platform.startswith("linux"):
```

Comment:
Do we still need to check the platform? I saw `get_current_node_accelerator_type` still checks it.

Reply:
won't need check platform. I will remove this.

Reply:
@jjyao please review new commits